cypress-tests/cypress/commands/apiCommands.js

```diff
@@ -125,13 +125,15 @@ Cypress.Commands.add(
   (slug = '',
     workspaceId = Cypress.env("workspaceId"),
     appId = Cypress.env("appId"),
-    componentSelector = "[data-cy='empty-editor-text']"
+    componentSelector = "[data-cy='widget-list-box-form']"
```

Comment:
@Srimanitejas123  Since it is a common function used along the appbuilder, we need to revert this and pass your selector param ```componentSelector```